src/main.cpp

```diff
@@ -692,10 +693,26 @@ Args parse_args(int argc, char* argv[])
   return args;
 }
 
+bool is_colorize()
+{
+  const char* color_env = std::getenv("BPFTRACE_COLOR");
+  if (!color_env) {
+    return isatty(STDOUT_FILENO) && isatty(STDERR_FILENO);
+  }
+
+  std::string mode(color_env);
```

Comment:
Nit: non-allocating view is probably better here
```suggestion
  std::string_view mode(color_env);
```

Reply:
Done